makefiles/Makefile.docs-charts

```diff
@@ -23,23 +29,23 @@ next-gen-html:
 	else \
 		exit 0; \
 	fi
-	rsync -az --exclude '.git' ${REPO_DIR}/../../snooty ${REPO_DIR}; \
-	cp ${REPO_DIR}/.env.production ${REPO_DIR}/snooty; \
+	rsync -az --exclude '.git' ${REPO_DIR}/../../snooty ${REPO_DIR};
+	cp ${REPO_DIR}/.env.production ${REPO_DIR}/snooty;
 	cd snooty; \
 	echo "GATSBY_SITE=${PROJECT}" >> .env.production; \
 	npm run build; \
 	cp -r "${REPO_DIR}/snooty/public" ${REPO_DIR};
   
 next-gen-stage: ## Host online for review
-	mut-publish public ${STAGING_BUCKET} --prefix="${MUT_PREFIX}" --stage ${ARGS}; \
+	mut-publish public ${STAGING_BUCKET} --prefix="integration/${MUT_PREFIX}" --stage ${ARGS}; \
```

Comment:
Is staging now under `integration/` for charts?

Reply:
doh! thank you for catching this!